### ISSUE TITLE
As a systemctl tool, should it itself be a systemctl unit?

### ISSUE THREAD

**Issue by Laotree** [shakg/g-systemctl#14]

**Comment by shakg** [comment]
Yep it makes sense, but I need to work on it with the new architecture. I've introduced docker + separate projects for backend and frontend with svelte. 

This new architecture will allow us the make some complex features easily like streaming the real time service logs to UI.
